sentry/src/main/java/io/sentry/Baggage.java

```diff
@@ -17,118 +25,149 @@ public final class Baggage {
 
   static final @NotNull String CHARSET = "UTF-8";
   static final @NotNull Integer MAX_BAGGAGE_STRING_LENGTH = 8192;
-  static final @NotNull Integer MAX_BAGGAGE_LIST_MEMBER_COUNT = 64;
```

Comment:
Since 64 is the lower limit of what we must propagate as per spec. If we specifically want to do it, I'm happy to add it back in. However, there's no mention of it in the docs (https://develop.sentry.dev/sdk/performance/dynamic-sampling-context/) and neither the javascript nor the python sdk check for the member count.

What would the value of adding the check for max list members? That we sometimes have a smaller payload if the member limit is hit before the character limit? This would also mean that we would not add our headers even if they could be added based on the character limit.

Reply:
If none of the other SDKs have it, we can leave it out as well. Thinking process was: it's the same as with character limit, if we go above the minimum requirement, we could cause other systems (proxy, etc.) to drop some of the payload that was already there before we added our things.

Reply:
After internal conversation: Let's add it back.

Reply:
We should add all third party list-items, regardless of count. We should just not add our items if that would make the count go above the limit.